testing/python/language/test_tilelang_language_nvf4_mma_block_scale.py

```
@@ -795,6 +1025,102 @@ def test_nvf4_mma_block_scale_varying_scale_correctness():
+
+    C = kernel(A, B, SFA, SFB)
+    ref = _reference_ue8m0_blockscaled_gemm(A, B, sfa_bytes, sfb_bytes, M, N, K, 32)
+    torch.testing.assert_close(C, ref, rtol=0.0, atol=0.0)
+
+
+def _cuda_toolkit_below(major: int, minor: int) -> bool:
+    try:
+        from tilelang.contrib import nvcc as _nvcc
+
+        return _nvcc.get_cuda_version() < (major, minor)
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)